docs/src/pages/components/tree-view/tree-view.md

```diff
@@ -63,6 +63,4 @@ const data = {
 
 The component follows the WAI-ARIA authoring practices.
 
-To have an accessible tree view:
-
-- You must use `aria-labelledby` or `aria-label` to reference or provide a label on the TreeView. Otherwise screen readers announce it as "tree", making it hard to understand the context of a specific treeitem for users who rely on screen readers.
+To have an accessible tree view you must use `aria-labelledby` or `aria-label` to reference or provide a label on the TreeView. Otherwise screen readers announce it as "tree", making it hard to understand the context of a specific tree item.
```

Comment:
I think you missed a couple 🙂 :

```suggestion
To have an accessible tree view you must use `aria-labelledby` or `aria-label` to reference or provide a label on the TreeView, otherwise screen readers will announce it as "tree", making it hard to understand the context of a specific tree item.
```

Reply:
Thanks :)